recipes/tinyobjloader/config.yml

```
@@ -0,0 +1,5 @@
+versions:
+  1.0.6:
+    folder: 1.x
+  2.0-rc1:
```

```suggestion
  "2.0-rc1":
```
due parser limitations